### ISSUE TITLE
ARIA Labels for Profile Email and Telephone

### ISSUE THREAD

**Issue by wmcconnell** [ASU-KE/UDS-WordPress-Theme#696]
**Issue**
Our in-theme Profile block is rendering generic ARIA label text, instead of applying the same text that is on-screen, and triggering alerts from Site Improve. Yes, this block should be deprecated in favor of Pitchfork People dynamic/static blocks, but we still have some around and we should make this simple update until profiles get settled across the board.

** Example **
A typical UDS Profile card will create output like this:

```
<ul class="person-contact-info">
	<li>
		<a href="mailto:gern.blanston@blanston.org" aria-label="Email Person Name" data-ga="gern.blanston@blanston.org" data-ga-name="onclick" data-ga-event="link" data-ga-action="click" data-ga-type="email" data-ga-region="main content" data-ga-section="profile">gern.blanston@blanston.org</a>
	</li>
	<li>
		<a href="tel:480 555 1212" aria-label="Call user" data-ga="480 555 1212" data-ga-name="onclick" data-ga-event="link" data-ga-action="click" data-ga-type="phone" data-ga-region="main content" data-ga-section="profile">480 555 1212</a>
	</li>
</ul>
```

The issue here is that the on-screen email address is `gern.blanston@asu.edu`, but the ARIA label is "Email Person Name"; the same issue occurs with the phone number, with the ARIA label just saying "Call user" instead of the actual phone number.

**Solution**
We just need to echo the same value for ARIA labels that we are already echo for displaying on-screen email addresses and phone numbers

[AB#43823](https://asuteam404.visualstudio.com/3c3feac4-94b8-464c-8938-c46f8a83b297/_workitems/edit/43823)

test new board push

AB#43825

**Comment by wmcconnell** [comment]
After doing a little research, it's not so clear what the ARIA labels here should be, if any. I need to double-check with Cindy regarding what SiteImprove is actually flagging, but I don't think we want to replicate the on-screen content (email address/phone number) in the label. We may just need more specific information, such as "Email Joe Smith" or "Call Joe Smith," as the screen readers will likely read out the link text.

More research in the meantime.
